### PR TITLE
Unified .va-tab-content styles. closes #3665

### LINKED ISSUE #3665: Unify .va-tab-content styles
We have `.va-tab-content` in `_va-tabs.scss` with a top `padding-top: 4rem` (soon to be 0rem), and again in `disability-benefits.scss` with `padding-top: 2em`. 

They have the same border style, so we just need to:

1. reconcile whether we actually need to have two different `padding-top` values
2a. If yes, add a `.disability-tab-content` class for the disability variant.
2b. If no, remove `.va-tab-content` from `disability-benefits.scss`

### PR DESCRIPTION
**Pull request by webinista**
- Removed .va-tab* styles from src/sass/disability-benefits.scss
- Added modules/va-tabs partial.

### CODE REVIEW

File: src/sass/disability-benefits.scss [hunk 1]
```diff
@@ -324,61 +325,6 @@ a.claim-list-item {
...
-  }
-
-  @media (min-width: $small-screen) {
-    padding-left: 4rem;
-    padding-right: 4rem;
-  }
-}
-
-.va-tab-content {
-  padding-top: 2em;
```
Comment (austin-martinez) [inline src/sass/disability-benefits.scss]: this padding is not present in `va-tabs.scss`

File: src/sass/disability-benefits.scss [hunk 2]
```diff
@@ -324,61 +325,6 @@ a.claim-list-item {
...
-    border-bottom-color: $color-white;
-  }
-
-  &:visited {
-    color: inherit;
-  }
-
-  @media (min-width: $small-screen) {
-    padding-left: 4rem;
-    padding-right: 4rem;
```
Comment (austin-martinez) [inline src/sass/disability-benefits.scss]: in `va-tabs.scss`, this value is 2rem

**Review by alexose — APPROVED** [review]
Other than the inconsistencies that @austin-martinez noticed, LGTM.

**Comment by webinista** [comment]
That's by design. I should have been clearer and more specific about the ask and goal here.

- Aim is to make `disability-benefits.scss` use `_va-tabs` and make that component be rendered consistently across our products.
- `_va-tabs` is also used on Prescriptions with the padding and margins from `_va-tabs`.
- This PR just swaps the old `va-tab*` styles with those from `_va-tabs`.

What I need is:

- A code review (completed here)
- A design review, which I assume requires a test account (I don't have one yet, and should have asked for beforehand ¯\_(ツ)_/¯ ).

**Comment by alexose** [comment]
@webinista If you just need to log in to see the changes (and maybe take screenshots), you could disable `RequiredLoginView` on your local install.  Although, that's kind of irritating... we should automate this somehow :)

**Comment by austin-martinez** [comment]
Here is a screenshot of the tab content without the top padding
screen shot 2016-12-20 at 5 27 50 pm

I think the tabs look fine with the smaller padding, but the content here is too close to the tabs, not sure what it looks like for Rx

File: src/sass/disability-benefits.scss (hunk 2 above)
Comment (webinista) [inline src/sass/disability-benefits.scss]: Yes. That should be removed for consistency with Rx tabs (design change requested in #3306/#3380). Happy to confirm with design.

**Comment by webinista** [comment]
Padding restored!

screen shot 2016-12-21 at 3 04 24 pm

**Review by melwoodard — APPROVED** [review]
LGTM from the screenshot. I second @alexose's comment on an easier way to do testing for login req'd apps - perhaps that's a ticket for the core team to address?

**Pull request merged**